cms/djangoapps/contentstore/views/tests/test_library.py

```diff
@@ -3,18 +3,33 @@
 
```

Comment:
I would prefer to move these new import/export tests to a different file since this file is specifically for testing `contentstore.views.library` - see line 2 of this file. Also, the new tests are not using the Django test client, which all the existing tests are - so they're a different type of test. Given that the new methods you're testing are `xmodule.modulestore.xml_exporter.export_library_to_xml` and `xmodule.modulestore.xml_importer. import_library_from_xml` I think the best place for these tests is `common/lib/xmodule/modulestore/tests/`.

You will also get a better coverage report when you move them there, since coverage for `/common/` code doesn't pick up tests that run from `/cms/`